src/app/src/main/kotlin/org/imperial/mrc/hint/clients/HintrAPIClient.kt

```diff
@@ -184,7 +184,7 @@ class HintrFuelAPIClient(
             return postEmpty("download/submit/${type}/${id}")
         }
 
-        val json = objectMapper.writeValueAsString(mapOf("state" to projectState))
+        val json = objectMapper.writeValueAsString(projectState)
```

Comment:
For clarity, might be worth calling this parameter something other than 'projectState' since it now contains both state and notes. "projectInfo" or "projectPayload"?